<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QDialog" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>202</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>285</width>
    <height>70</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2850</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>73</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_5">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>96</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>261</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="minimumSize">
     <size>
      <width>45</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="scrGreen">
    <property name="minimumSize">
     <size>
      <width>215</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:green;</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>261</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="minimumSize">
     <size>
      <width>45</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="scrBlue">
    <property name="minimumSize">
     <size>
      <width>215</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: Blue;</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>15</y>
     <width>267</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="minimumSize">
     <size>
      <width>181</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>181</width>
      <height>23</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>BackGround Color</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>131</width>
      <height>23</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blue</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>149</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="btnOk">
    <property name="minimumSize">
     <size>
      <width>72</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>72</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="minimumSize">
     <size>
      <width>72</width>
      <height>23</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>72</width>
      <height>23</height>
     </size>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>261</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="minimumSize">
     <size>
      <width>45</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="scrRed">
    <property name="minimumSize">
     <size>
      <width>215</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 4);</string>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
